requirements/dev-extra.in

```diff
@@ -0,0 +1,2 @@
+tox
+tox-battery
```

Comment:
Maybe we should pin these and/or enumerate the indirect dependencies until we get the pip-tools issue fixed?